lib/emis/responses/get_deployment_response.rb

```diff
@@ -5,12 +5,16 @@
 
 module EMIS
   module Responses
+    # EMIS get deployments response
     class GetDeploymentResponse < EMIS::Responses::Response
+      # (see EMIS::Responses::GetCombatPayResponse#item_tag_name)
       def item_tag_name
         'deployment'
       end
 
       # rubocop:disable Metrics/MethodLength
+
+      # (see EMIS::Responses::GetCombatPayResponse#item_schema)
```

Comment:
Perhaps use the [`@see`](https://www.rubydoc.info/gems/yard/file/docs/Tags.md#see) tag for all of these?

Reply:
using `(see)` copies the documentation from the other method, seems easier than linking to it